util/schema-gen/src/org/javarosa/xform/schema/FormInstanceLoader.java

```diff
@@ -0,0 +1,80 @@
+package org.javarosa.xform.schema;
+
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.model.instance.FormInstance;
+import org.javarosa.core.model.instance.InstanceInitializationFactory;
+import org.javarosa.core.model.instance.TreeElement;
+import org.javarosa.core.model.instance.TreeReference;
+import org.javarosa.form.api.FormEntryModel;
+import org.javarosa.xform.parse.XFormParseException;
+import org.javarosa.xform.parse.XFormParser;
+import org.javarosa.xform.util.XFormUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * Logic for loading a particular xml form instance into a FormDef.
+ *
+ * @author Phillip Mates (pmates@dimagi.com)
+ */
+public class FormInstanceLoader {
+    private static InstanceInitializationFactory iif;
```

Comment:
No idea, it was unset in FormLoaderTask: https://github.com/dimagi/commcare-odk/blob/master/app/src/org/odk/collect/android/tasks/FormLoaderTask.java#L170

It looks like in this particular case the XFormPlayer already does the iif setup by calling XFormEnvironment.setup() and using https://github.com/dimagi/javarosa/blob/master/util/schema-gen/src/org/javarosa/engine/MockupProviderFactory.java